VirtoCommerce/vc-platform
# return back to current state after password window closed

How to reproduce issue:
1. Login and close new password window
2. Go to Store blade for example (http://admin/#/workspace/store)
3. Refresh browser page
4. You see "new password window" and when you click 'postpone' button browser redirect to home page, instead to redirect to store blade (http://admin/#/workspace/store)
